retrolab/app.py

```diff
@@ -63,6 +63,9 @@ def get_page_config(self):
             "mathjax_url",
             "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/MathJax.js",
         )
+        if not url_is_absolute(mathjax_url) and not mathjax_url.startswith(self.base_url):
```

Comment:
Looks like they could be a better fix than this, but this seems to be fine for now.

Happy to improve that in follow-up PRs.